source/style-guide/style/copyrights.txt

```diff
@@ -1,9 +1,21 @@
 .. _copyrights:
+.. _trademarks:
+.. _third-party-names-and-trademarks:
 
-==========
-Copyrights
-==========
+=========================
+Copyrights and Trademarks
+=========================
 
-.. include:: /style-guide/includes/page-needs-update.rst
+- **Don't** include copyright or trademark symbols after a trademark or
+  trade name.
```

Comment:
I wouldn't include this.

Reply:
**question:** Why not? He did say that we don't need to do it.

Reply:
And this sticks more closely to his written guidance: only talk about logos.

Reply:
Updated.